app/views/tasks/_form.html.erb

```diff
@@ -0,0 +1,9 @@
+<%= form_with(model: @task, local: true) do |f|%>
+  <%= f.label :title %>
```

Comment:
There's a lot of duplicate code on this form and the edit_form. Please make this DRYer